.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
```

Comment:
is this going to potentially introduce code behaviour changes, if a dev doesn't carefully check what they are committing?

prettier on save is safer, because usually it doesn't modify the compiled JS